### PR TITLE
Water bottles (and other closed non-carbonated drinks) no longer explode when shaken

### PR DESCRIPTION
**Pull request by JohnWildkins**
title
also adds a tag to reagents to track if they're carbonated or not

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

**Comment by Mwahahahaha** [comment]
Finally this menace has been stopped.
No-one has to live in fear. There is a ray of sunshine after all.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -37,6 +37,7 @@
 	//Never ever ever ever change this value for datum/reagent. This should only be used for massive, yet specific things like drinks or food where it is infeasible to assign a specific heat value.
 
 	var/germ_adjust = 0 // for makeshift bandages/disinfectant
+	var/carbonated = FALSE // if it's carbonated or not
```
Comment (MarinaGryphon) [inline code/modules/reagents/Chemistry-Reagents.dm]: holy shit it's per-reagent? i love you

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 2]
```diff
@@ -29,9 +29,11 @@
 			playsound(loc,'sound/items/soda_shaking.ogg', rand(10,50), 1)
 			return
 		if(shaken)
-			boom(user)
-		else
-			open(user)
+			for(var/datum/reagent/R in reagents.reagent_list)
+				if(R.carbonated)
```
Comment (MarinaGryphon) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: YEEEEEEEEESSSSSSSSSSs

**Pull request merged**